AnnoDesigner/Models/LayoutObject.cs

```diff
@@ -167,6 +198,33 @@ public Point Position
             }
         }
 
+        public double BlockedAreaLength => WrappedAnnoObject.BlockedAreaLength;
+
+        public double BlockedAreaWidth
+        {
+            get
+            {
```

Comment:
Is it possible to "cache" the calculated value? I also don't know if it would be beneficial.

Reply:
I'll cache the `BlockedAreaScreenRect` instead